conjur/argument_parser/_list_parser.py

```diff
@@ -0,0 +1,61 @@
+import argparse
+from conjur.argument_parser.parser_utils import command_description, command_epilog, formatter, title_formatter
+
+
+class ListParser:
+    def add_list_parser(self):
+        """
+        Method adds list parser functionality to parser
+        """
+        list_subparser = self._init_list_parser()
+        ListParser.add_list_options(list_subparser)
```

Comment:
why isn't this self?

Reply:
static function but still could be self
changing

Reply:
curious though why you chose `ListParser` to start with. Wass there a particular reason?